apis/v1beta1/vspherefailuredomain_webhook.go

```diff
@@ -80,14 +76,3 @@ func (r *VSphereFailureDomain) ValidateUpdate(old runtime.Object) error {
 func (r *VSphereFailureDomain) ValidateDelete() error {
 	return nil
 }
-
-// Default implements webhook.Defaulter so a webhook will be registered for the type.
-func (r *VSphereFailureDomain) Default() {
```

Comment:
why are we removing the defaulting. This is just the deprecation announcement

Reply:
The broader idea behind the PR was:
1. To add the depreciation notice to the API and
2. To remove the functionality exercised when setting the flag 

According to the PR, irrespective of what value the user sets, no resource tagging gets executed.